.github/workflows/phpunit-tests.yml

```diff
@@ -124,14 +124,15 @@ jobs:
   # - Submit the test results to the WordPress.org host test results.
   # - todo: Configure Slack notifications for failing tests.
   test-php:
-    name: ${{ matrix.php }}${{ matrix.memcached && ' with memcached' || '' }} on ${{ matrix.os }}
+    name: ${{ matrix.php }}${{ matrix.multisite && ' multisite' || '' }}${{ matrix.memcached && ' with memcached' || '' }} on ${{ matrix.os }}
     needs: setup-wordpress
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        php: [ '8.0', '7.4', '7.3', '7.2', '7.1', '7.0', '5.6.20' ]
+        php: [ '5.6.20', '8.0', '7.4', '7.3', '7.2', '7.1', '7.0' ]
         os: [ ubuntu-latest ]
         memcached: [ false ]
+        multisite: [ false, true ]
         include:
           # Include job for PHP 7.4 with memcached.
```

Comment:
Can we add `multisite: false` to these individually defined jobs so that it's more clear on quick glance?

Also, with this new configuration, multisite will not be tested using memcached. We should include a multisite memcache job in this list using the latest PHP.

Reply:
> with this new configuration, multisite will not be tested using memcached

Good spot! I missed that.